mmv1/products/logging/Metric.yaml

```diff
@@ -240,28 +231,19 @@ properties:
         properties:
           - !ruby/object:Api::Type::Integer
             name: numFiniteBuckets
-            at_least_one_of:
-              - bucket_options.0.exponential_buckets.0.num_finite_buckets
-              - bucket_options.0.exponential_buckets.0.growth_factor
-              - bucket_options.0.exponential_buckets.0.scale
             description: |
               Must be greater than 0.
+            required: true
           - !ruby/object:Api::Type::Double
             name: growthFactor
-            at_least_one_of:
-              - bucket_options.0.exponential_buckets.0.num_finite_buckets
-              - bucket_options.0.exponential_buckets.0.growth_factor
-              - bucket_options.0.exponential_buckets.0.scale
             description: |
               Must be greater than 1.
+            required: true
```

Comment:
just making sure, are _all_ of these fields required? Or only `numFiniteBuckets`?

Reply:
Yes. Thanks!